client/state/posts/selectors/get-post-revision-fields.js

```diff
@@ -0,0 +1,9 @@
+import { createSelector } from '@automattic/state-utils';
+import { get } from 'lodash';
+
+export const getPostRevisionFields = createSelector(
+	( state, siteId, postId ) => {
+		return get( state.posts.revisions.diffs, [ siteId, postId, 'revisionFields' ] );
```

Comment:
lodash is discouraged. You can get the same behavior with native JS:
```suggestion
		return state.posts.revisions.diffs?.[ siteId ]?.[ postId ]?.revisionFields;
```

Reply:
Fair enough :D. I've pushed an update that removes the heading